Demo/src/main/java/com/braintreepayments/demo/PayPalFragment.java

```diff
@@ -69,13 +74,13 @@ private void launchPayPal(boolean isBillingAgreement) {
                     dataCollector.collectDeviceData(activity, (deviceData, dataCollectorError) -> mDeviceData = deviceData);
                 }
                 if (isBillingAgreement) {
-                    payPalClient.requestBillingAgreement(activity, getPayPalRequest(null), payPalError -> {
+                    payPalClient.requestBillingAgreement(activity, createPayPalVaultRequest(activity), payPalError -> {
```

Comment:
Should we update the demo to use `tokenizePayPalAccount` since we're deprecating `requestBillingAgreement` and `requestOneTimePayment`?

Reply:
Good call - we updated the `PreferredPaymentMethodsFragment` and `PayPalFragment` to use `tokenizePayPalAccount`